README.md

```diff
@@ -113,6 +113,9 @@ achieved by creating Feature Services:
 - Share browser APIs and resources not intended for shared use (e.g.
   [History][history-service-pkg], LocalStorage).
 - Share configuration across Feature Apps, but only maintain it once.
+- Provide a [versioned API](#feature-service-instantiation--api-versioning) to
+  allow backward compatibility and thus flexible and decoupled deployments of
+  different consumers.
```

Comment:
In this list we wanted to give the reasons why you should create a feature service. Is this point alone a reason to do so, or is it more of an overarching benefit/requirement?

Reply:
i think its also a reason